tasks/src/fill-mask/data.ts

```diff
@@ -28,24 +28,24 @@ const taskData: TaskData = {
 				"type": "chart",
 				data: [
 					{
-						"label": "life",
-						"score": 0.10933306068181992
+						"label": "wolf",
+						"score": 0.487
 					},
 					{
-						"label": "survival",
-						"score": 0.039418820291757584
+						"label": "dog",
+						"score": 0.061
 					},
 					{
-						"label": "love",
-						"score": 0.032930586487054825
+						"label": "cat",
+						"score": 0.058
 					},
 					{
-						"label": "freedom",
-						"score": 0.03009609691798687
+						"label": "fox",
+						"score": 0.047
 					},
 					{
-						"label": "simplicity",
```

Comment:
I didn't want to touch output so people would know what they expect but it's easier on the eyes.